shelley/chain-and-ledger/shelley-spec-ledger-test/src/Test/Shelley/Spec/Ledger/Utils.hs

```diff
@@ -123,14 +126,15 @@ import Test.Tasty.HUnit
   ( Assertion,
     (@?=),
   )
+import Test.QuickCheck ( Arbitrary )
 import Shelley.Spec.Ledger.STS.Utxow (UtxowPredicateFailure, UTXOW)
 import Shelley.Spec.Ledger.STS.Utxo (UtxoPredicateFailure, UTXO)
 import Shelley.Spec.Ledger.STS.Deleg (DELEG, DelegPredicateFailure)
 import Shelley.Spec.Ledger.STS.Delegs (DELEGS, DelegsPredicateFailure)
 
-type ShelleyTest era =
-  ( ShelleyBased era,
-    Core.Value era ~ Coin,
+type ShelleyTest era = (ShelleyBased era, Arbitrary (Core.Value era),
+    HedGen (Core.Value era), Split (Core.Value era), P.PartialOrd (Core.VALUE era),
+    Arbitrary (Core.VALUE era), HedGen (Core.VALUE era), Split (Core.VALUE era) )
```

Comment:
remove partial order and add a a constraint that requires to have `pointwise` comparison

Reply:
removed